src/libraries/System.Web.HttpUtility/src/System/Web/Util/UriUtil.cs

```diff
@@ -51,17 +43,17 @@ internal static bool TrySplitUriForPathEncode(string input, [NotNullWhen(true)]
                     if (authorityIndex != -1)
```

Comment:
```suggestion
                    if (authorityIndex >= 0)
```
Comparison to `0` is better for machine code, as the CPU can fuse the comparison and the jump.